dev_guide/deployments.adoc

```diff
@@ -273,6 +273,14 @@ image stream changes and the new tag value differs from the current image
 specified in the deployment configuration's `helloworld` container, a new
 deployment is created using the new tag value for the `helloworld` container.
 
+The following table explains the possible combinations between ConfigChange
+and ImageChange triggers.
+
+  Trigger                  | true                                      | false
+  -----------------------  | ----------------------------------------- | ------------------------------------------------------
+  ConfigChange             | Deployments are started automatically     | Deployments need to be started manually
+  ImageChange (automatic)  | Images are always resolved automatically  | Only the initial image will be resolved in the template
```

Comment:
Actually this behavior has always been desirable. The controllers should wait on the image the ICT points to, so that we won't try to deploy things that don't exist. ICTs are a convenient way of notifying for image updates in the cluster. If you have no use for it, you shouldn't have it on your DC.